src/tile.cpp

```diff
@@ -1528,7 +1516,7 @@ void Tile::setTileFlags(const Item* item)
 		setFlag(TILESTATE_BED);
 	}
 
-	const Container* container = item->getContainer();
+	auto container = item->getContainer();
```

Comment:
Could this be `const auto`
```suggestion
	const auto container = item->getContainer();
```